### PR TITLE
Fix link errors in ui_fw_api and tizen-studio

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

Fix the issues under ui_fw_api and Tizen Studio, from #1401 

### Bugs Fixed ###

 - Issue https://github.com/Samsung/tizen-docs/issues/1401

### CODE REVIEW

**Comment by mijin80lee** [comment]
LGTM, thank you!

File: docs/application/web/api/6.0/ui_fw_api/Wearable_UIComponents/wearable_drawer.htm [hunk 1]
```diff
@@ -232,7 +232,7 @@ <h2><a id="options-list"></a>Options</h2>
 	</tbody>
 </table>
 
-<h2><a href="#options-example"></a>HTML Example with Options</h2>
+<h2><a id="options-example"></a>HTML Example with Options</h2>
 <h3>Drawer positioning</h3>
 You can declare to 'position' option as below method.
```
[the remark this answers is not in the document]
Comment (annie-samsung) [reply docs/application/web/api/6.0/ui_fw_api/Wearable_UIComponents/wearable_drawer.htm:237]: @mijin85cho, could you check the above query and clarify?

[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/web/api/6.0/ui_fw_api/Wearable_UIComponents/wearable_drawer.htm:237]: Web API pages follows different style with developer guide pages. So we can keep current.

[the remark this answers is not in the document]
Comment (annie-samsung) [reply docs/application/web/api/6.0/ui_fw_api/Wearable_UIComponents/wearable_drawer.htm:237]: @mijin85cho, thanks for clarifying.

**Review by annie-samsung — APPROVED** [review]
approving

**Pull request merged**